vendor/github.com/Microsoft/hnslib/hcn/hcnglobals.go

```diff
@@ -90,6 +90,7 @@ var (
 	// HNS 15.5+ allows for Modify Loadbalancer support in Zn
 	ModifyLoadbalancerVersion = VersionRanges{
 		VersionRange{MinVersion: Version{Major: 15, Minor: 5}, MaxVersion: Version{Major: 15, Minor: math.MaxInt32}},
```

Comment:
this is the bugfix? `Major: 15` should have been `Major: math.MaxInt32` like the other entries?